src/js/components/notification.js

```diff
@@ -27,6 +27,19 @@ export class SingleNotification extends React.Component {
     this.props.markNotification(this.props.notification.id);
   }
 
+  getUrgency() {
+    switch (this.props.notification.subject.urgency) {
```

Comment:
I know this is think day type work but I like to discuss stuff like code cleanliness, design, and other such things. My comments here are more for conversation than action. Unless you see the value in what I'm saying and feel it's worth the change. And of course if you disagree then that makes for better conversation.

First thing... I (almost irrationally) **_HATE_**  switch statements. I tend to prefer a pattern of using a  object literal with the property names equal to the case values and have the value for that property be either the string you want like you have here or better, in my opinion, a function that will have a common signature that with the other functions on that object. I think it makes for a much more flexible piece of code. Now you could probably get all that with case statement and discipline but I think you definitely give up something.

Again.. just talking about the structure in general. Not specifically this case (seewhatididthere).